### PR TITLE
Admin user creation method

### PR DESCRIPTION
**Pull request by abhi36**
Adding method to create a user using AdminCreateUser

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +43,10 @@ before running them.
 Version History
 ---------------
 
+0.2.11 (06/01/2020)
+
+Added method to get a user by an access token - bjoernHeneka
+
```
Comment (pmill) [inline README.md:49]: Can you format this like the other lines (indented and link your username)

File: examples/getUserByToken.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+$authenticationResponse = $client->authenticate($username, $password);
+$accessToken = $authenticationResponse['AccessToken'];
+
+try {
+    $user = $client->getUserByToken($accessToken);
+    echo $user['Username'].PHP_EOL;
+    var_dump($user['UserAttributes']);
+} catch (Exception $e) {
+    echo "An error occurred: ".$e->getMessage();
```
Comment (pmill) [inline examples/getUserByToken.php:17]: Can you add a space before and after the `.`? Same for the `echo` line above

File: README.md (hunk 1 above)
Comment (pmill) [reply README.md:49]: Also this has bjoernHeneka's username, please change that to your own.

**Review by pmill — CHANGES_REQUESTED** [review]
Hi @abhi36, thanks for your PR, just a few changes please.

The PR title doesn't seem to relate to the changes made, can you update it?

Also can you rebase your branch? There are a few commits in there that don't look related.

File: README.md (hunk 1 above)
Comment (abhi36) [reply README.md:49]: @pmill Could you please give me access to push my branch? My this merge request can be ignored

Comment (pmill) [reply README.md:49]: You just need to push to your own branch on your fork and the PR will be updated.

Comment (abhi36) [reply README.md:49]: Thanks & done :)

**Comment by pmill** [comment]
Closing this as you seem to have replaced it with #39

**Pull request closed without merging**